addons/knobs/package.json

```diff
@@ -37,7 +37,8 @@
     "react-dom": "^15.5.4",
     "style-loader": "^0.17.0",
     "typescript": "^2.2.2",
-    "typescript-definition-tester": "^0.0.5"
+    "typescript-definition-tester": "^0.0.5",
```

Comment:
do we still need this, now that we have moved the typescript definitions?

- [ ] @joscha to follow up